q2_qemistree/_itol_metadata.py

```diff
@@ -19,6 +22,12 @@ def classyfire_to_colors(classified_feature_data: pd.DataFrame,
     annotations = classified_feature_data[classyfire_level].unique()
     colors = sns.color_palette(color_palette,
                                n_colors=len(annotations)).as_hex()
+
+    # give a heads up to the user
+    if set(colors) < len(annotations):
+        warnings.warn(UserWarning, "The mapping between colors and annotations"
+                      " is not unique, some colors have been repeated")
```

Comment:
I think this should be:
```suggestion
warnings.warn("The mapping between colors and annotations"
              " is not unique, some colors have been repeated", UserWarning)
```